blimflix/client/app/providers/favorites.factory.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+  'use strict';
+
+  /**
+   * Chain to fetch module
+   *   https://github.com/johnpapa/angular-styleguide#style-y022
+   */
+  angular
+    .module('blimFlix.providers')
+    .factory('favorites', favorites);
+    favorites.$inject = ['$resource'];
+
+  /**
+   * Avoid anonymous functions as callbacks
+   *   https://github.com/johnpapa/angular-styleguide#style-y024
+   */
+  /* @ngInject */
+  function favorites($resource) {
+    /**
+     * Accessible members at top
+     *   https://github.com/johnpapa/angular-styleguide#style-y052
+     */
+    return $resource('http://localhost:3333/users/:user_id/favorites/:movie_id',
+      {user_id: '@user_id', movie_id: '@movie_id'},
+      {
+        'get'   : { method: 'GET', isArray: true, },
```

Comment:
could be better use the same convention for all the resourses
the standard convention is
```
query : GET an array of elements
get : GET just one of these elements
save: POST create a new element
delete: DELETE one of these elements
remove: same as delete
update: PUT update one of these elements
```